teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'unknown', parent_permission: true)
+    super(name, age, parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services
+    true
+  end
```

Comment:
- [ ] Please kindly add the omitted `?` to this method, in accordance to Ruby style convention; it indicates that a method returns a Boolean value (true or false) or an object to indicate a true value (or “truthy” value). You can reference [this for further clarification](https://hackernoon.com/understanding-boolean-methods-in-ruby-with-examples-363n36nu).